managed/services/supervisord/supervisord.go

```diff
@@ -772,7 +774,7 @@ command =
         cfg:default.auth.generic_oauth.auth_url="{{ .PerconaSSODetails.IssuerURL }}/authorize"
         cfg:default.auth.generic_oauth.token_url="{{ .PerconaSSODetails.IssuerURL }}/token"
         cfg:default.auth.generic_oauth.api_url="{{ .PerconaSSODetails.IssuerURL }}/userinfo"
-		cfg:default.auth.generic_oauth.role_attribute_path="contains(portal_admin_orgs[*], '{{ .PerconaSSODetails.OrganizationID }}') && 'Admin' || 'Viewer'"
+		cfg:default.auth.generic_oauth.role_attribute_path="(contains(portal_admin_orgs[*], '{{ .PerconaSSODetails }}') || contains(pmm_demo_ids[*], '{{ .PMMServerID }}')) && 'Admin' || 'Viewer'"
```

Comment:
should the `.OrganizationID` be removed here?

Reply:
No, bc the `PMMServerID` will appear only in profiles of users that connected the _temporary Demo instances_. The normal instances will use the old way to map roles.

Reply:
Agree, but you removed `.OrganizationID`

Reply:
Oh, thanks, I didn't notice that. That's why my tests are failing 😄